### PR TITLE
Tweaks in some mobs

### PR DESCRIPTION
**Pull request by jotite19**
Buff Benis, Nerf uibuta, Nerf uibuta spawns (to much hipos)

### CODE REVIEW

File: servers/minecraft/plugins/Mobzy/spawns.yml [hunk 1]
```diff
@@ -139,7 +139,8 @@ regions:
         max-amount: 4
         radius: 40
         max-local-group: 4
-        local-group-radius: 40
+        local-group-radius: 80
+        priority: 0.3
```
Comment (Boy0000) [inline servers/minecraft/plugins/Mobzy/spawns.yml:143]: again, it doesn't break anything  but neither does it actually hinder them from spawning. Only works for flying mobs

File: servers/minecraft/plugins/Mobzy/spawns.yml [hunk 2]
```diff
@@ -297,7 +298,7 @@ regions:
 
       - mob: Hashofu
         min-light: 5
-        max-amount: 4
+        max-amount:4
```
Comment (Boy0000) [inline servers/minecraft/plugins/Mobzy/spawns.yml:301]: missed a space there

**Review by Boy0000 — CHANGES_REQUESTED** [review]

File: servers/minecraft/plugins/Mobzy/spawns.yml (hunk 1 above)
Comment (Boy0000) [reply servers/minecraft/plugins/Mobzy/spawns.yml:143]: regarding the priority

**Review by Boy0000 — APPROVED** [review]

File: servers/minecraft/plugins/Mobzy/spawns.yml [hunk 3]
```diff
@@ -171,12 +171,15 @@ regions:
   - name: Inverted Forest
     spawns:
       - reuse: Template:Benikuchinawa
-        max-local-group: 2
-        local-group-radius: 200
+        max-local-group: 4
```
Comment (Boy0000) [inline servers/minecraft/plugins/Mobzy/spawns.yml:174]: This is already the main place people farm splitjaws. So if anything it should be lowered.

**Review by Boy0000 — CHANGES_REQUESTED** [review]

**Comment by jotite19** [comment]
The thing is that it got lowered recently if im not mistaken and they became rarer.

**Comment by Boy0000** [comment]
yeah, layer 3 is their main area, so above that they are rare

**Review by Boy0000 — APPROVED** [review]

**Pull request merged**